bootloader/src/rtl872x/rtl_it.c

```diff
@@ -128,6 +143,30 @@ void UsageFault_Handler(void) {
     Fault_Handler(UsageFault);
 }
 
+void SecureFault_Handler_NS(void) {
```

Comment:
As per the map file and IDAU configuration `10006318` seems to reside in secure region.
```
10006318 <SecureFault_Handler_NS>:
    /* Go to infinite loop when Usage Fault exception occurs */
    Fault_Handler(UsageFault);
}
```

```
const TZ_CFG_TypeDef tz_config[]= {
    //  Start                    End                NSC
    {0x40000000,            0x50000000-1,            0},  // entry0: Peripherals NS
    {0x1010A000,            0x101D4000-1,            0},  // entry1: IROM & DROM NS
    {0x100E0000,            0x10100000-1,            0},  // entry2: BT/WIFI Extention SRAM
    {0x00000000,            0x08060000-1,            0},  // entry3: KM0 ROM, KM0 SRAM, Retention SRAM, PSRAM, KM0 FLASH
    {0x08060000,            0x1007C000-1,            0},  // entry4: KM4 Flash, KM4 SRAM
    {0xFFFFFFFF,            0xFFFFFFFF,              0},  // entry5: None
    {0xFFFFFFFF,            0xFFFFFFFF,              0},  // entry6: None
    {0xFFFFFFFF,            0xFFFFFFFF,              0},  // entry7: None
};
```
<img width="753" alt="Screenshot 2023-09-22 at 11 39 36" src="https://github.com/particle-iot/device-os/assets/7424522/9ff67265-7166-4ad2-9b2d-265d2acb0e6b">

Reply:
It applies to `{0x08060000,            0x1007C000-1,            0},  // entry4: KM4 Flash, KM4 SRAM` entry, which is marked as NS by IDAU.

Reply:
Oh, I missed that.